### DISCUSSION TITLE
https://maxkim-j.github.io/posts/async-console-log

### DISCUSSION THREAD

**Discussion by MaxKim-J** [MaxKim-J/maxkim-j.github.io#86] [Announcements]
#          🛹 console.log 문제 풀이 - 마이크로태스크 큐      

https://maxkim-j.github.io/posts/async-console-log

**Comment by MaxKim-J** [comment]
<p>머리 아픈 내용인데 예제도 굉장히 참신하고 (ㅋㅋㅋ) 쉽고 자세하게 설명해주셨네요! 잘 읽었습니다. 감사합니다 (_ _)</p>

>_This comment was made by Disqus. 2020-10-03 11:10:44 Hannah Lee(disqus_2El8ZH5iYw)_

**Comment by MaxKim-J** [comment]
<p>참신하다고 느끼셨다니 다행입니다ㅎㅎ 읽어주셔서 감사합니다!!</p>

>_This comment was made by Disqus. 2020-10-04 12:20:18 Max Kim(disqus_O30MHz2G2q)_

**Comment by hon9g** [comment]
정답이 숨겨져있군요
